src/Iot.Device.Bindings/Iot.Device.Bindings.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="System.IO.Ports" Version="$(SystemIOPortsPackageVersion)" />
     <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj" />
     <PackageReference Include="UnitsNet" Version="$(UnitsNetPackageVersion)" />
+    <PackageReference Include="System.Management" Version="5.0.0-alpha.1.19618.1" />
```

Comment:
I think there should be a new version out now. I wrote this binding some time ago.

Reply:
should this be Windows-only dependency? @joperezr for how to do it

Reply:
@Ellerbach While I can see that there are newer versions available on the dev servers, I cannot seem to use them. Possibly because none of the never versions have been published to the official nuget feed.

Reply:
Now switched to the latest version that seems to install directly. I expect a final version to be uploaded to nuget.org when .NET 5.0 ships.